topics/metagenomics/tutorials/beer-data-analysis/tutorial.md

```diff
@@ -0,0 +1,391 @@
+> - this sequence has been sequenced with a quality `$'())#$$%#$%%'-$&$%'%#$%('+;<>>>18.?ACLJM7E:CFIMK<=@0/.4<9<&$007:,3<IIN<3%+&$(+#$%'$#$.2@401/5=49IEE=CH.20355>-@AC@:B?7;=C4419)*$$46211075.$%..#,529,''=CFF@:<?9B522.(&%%(9:3E99<BIL?:>RB--**5,3(/.-8B>F@@=?,9'36;:87+/19BAD@=8*''&''7752'$%&,5)AM<99$%;EE;BD:=9<@=9+%$`.
+>
+> But what does this quality score mean?
+>
+> The quality score for each sequence is a string of characters, one for each base of the nucleic sequence, used to characterize the probability of mis-identification of each base. The score is encoded using the ASCII character table (with [some historical differences](https://en.wikipedia.org/wiki/FASTQ_format#Encoding)):
+>
+> ![Encoding of the quality score with ASCII characters for different Phred encoding. The ascii code sequence is shown at the top with symbols for 33 to 64, upper case letters, more symbols, and then lowercase letters. Sanger maps from 33 to 73 while solexa is shifted, starting at 59 and going to 104. Illumina 1.3 starts at 54 and goes to 104, Illumina 1.5 is shifted three scores to the right but still ends at 104. Illumina 1.8+ goes back to the Sanger except one single score wider. Illumina](./images/fastq-quality-encoding.png)
+>
+> So there is an ASCII character associated with each nucleotide, representing its [Phred quality score](https://en.wikipedia.org/wiki/Phred_quality_score), the probability of an incorrect base call:
+>
+> Phred Quality Score | Probability of incorrect base call | Base call accuracy
+> --- | --- | ---
+> 10 | 1 in 10 | 90%
+> 20 | 1 in 100 | 99%
+> 30 | 1 in 1000 | 99.9%
+> 40 | 1 in 10,000 | 99.99%
+> 50 | 1 in 100,000 | 99.999%
+> 60 | 1 in 1,000,000 | 99.9999%
+>
+{: .details }
+
+# Data quality
+
+## Assess data quality
+
+Before starting to work on our data, it is necessary to assess its quality. This is an essential step if we aim to obtain a **meaningful downstream analysis**.
+
+**FastQC** is one of the most widely used tools to **check the quality** of data generated by High Throughput Sequencing (HTS) technologies.
+
+> ### {% icon hands_on %} Hands-on: Quality check
+>
+> 1. {% tool [FASTQC](toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.73+galaxy0) %} with the following parameters
+>    - {% icon param-files %} *"Raw read data from your current history"*: `Reads`
+>
+> 2. Inspect the generated HTML file
+>
+{: .hands_on}
+
+**FastQC** provides information on various parameters, such as the range of quality values across all bases at each position:
+
+![FastQC Per base sequence quality with scores below 200](./images/fastqc_1.png "Per base sequence quality")
+
+We can see that the quality of our sequencing data grows after the first few bases, stays around a score of 18, which is a relatively low value compared to other sequencing technologies, and then decreases again at the end of the sequences.
+
```

Comment:
Maybe we can mention the QC tutorial here, for people who want more information about the FASTQC plots, what do you think/.